### ISSUE TITLE
OpenVPN 2.6_rc1 install moves all configs to config-auto

### ISSUE THREAD

**Issue by cron2** [OpenVPN/openvpn-build#312]
Hi,

this might be something particular to my systems, but since it has happened twice now, I decided to call it a bug.

- Upgrade from an older OpenVPN installation (2.5 something, maybe even older) to 2.6_rc1
- GUI starts, declares "there are no config profiles" (in german, so not sure what the original English text would be)
- when looking, all .ovpn files got moved to ``c:\program files\openvpn\config-auto`` and ``...\config`` is empty except for "README.txt" and "where are my config files.txt".

Both systems are Win10, and have been uprgaded from Win7 at some point.  One is a VM which is used for openvpn windows testing, so has seen a number of NSIS (!!) installations, built by me on Ubuntu/MinGW.  The other is a Laptop, and tbh I have no idea if the previous version of OpenVPN was NSIS or MSI installed.

Maybe put this under "upgrading from a NSIS-installed OpenVPN to a MSI-installed moves all configs to config-auto/"?  If it happens MSI to MSI, someone else would have complained already...

**Comment by selvanair** [comment]
For NSIS to MSI, the intent is to move config folder profiles to config-auto **only** if OpenVPNService is running. The assumption here is that NSIS installation has to be 2.4 or older[*] and all profiles in config folder are autostarted in those versions if the service is running. 
Looks like the recent change to always install OpenVPNService might have messed up with this logic.

[*] Although I do cross-compile, I have not been using openvpn-build and thus NSIS for long, so have not noticed this. I cross-compile from within the git sources using a tiny script and copy the executables to Windows for testing. Once dependency dlls are in place, this is much easier and quicker than the old openvpn-build/generic setup.

**Comment by lstipakov** [comment]
Should be solved here https://github.com/OpenVPN/openvpn-build/pull/316

**Comment by cron2** [comment]
Hi,

On Tue, Jan 10, 2023 at 06:40:35AM -0800, Lev Stipakov wrote:
> Should be solved here https://github.com/OpenVPN/openvpn-build/pull/316

I rolled back my Win10 VM to "pre upgrade" and there was a 2.6_git
version with my own NSIS installer on it.

"...\config\" directory present, no "...\config-auto\".

Upgraded using Lev's test installer from

  https://github.com/lstipakov/openvpn-build/actions/runs/3882972481

after that, I have a 2.6_rc1 OpenVPN, and a "...\config-auto\" directory,
but all the existing configs are still in "...\config\" - so things work
out of the box.

So, this installer does fix the issues I experienced.

gert


-- 
"If was one thing all people took for granted, was conviction that if you 
 feed honest figures into a computer, honest figures come out. Never doubted 
 it myself till I met a computer with a sense of humor."
                             Robert A. Heinlein, The Moon is a Harsh Mistress

Gert Doering - Munich, Germany                             ***@***.***
